packages/elements/src/card/index.ts

```diff
@@ -170,6 +158,15 @@ export class Card extends BasicElement {
     this.footerHasContent = isSlotEmpty(event.target as HTMLSlotElement);
   }
 
+  /**
+   * Run on overlay menu open changed
+   * @param event overlay menu opened changed event
+   * @returns {void}
+   */
+  private onMenuOpenChanged (event: OpenedChangedEvent): void {
+    this.openMenuElement?.setAttribute('aria-expanded', String(event.detail.value));
```

Comment:
Do not use `setAttribute` on internal elements. Use `render` lifecycle

Reply:
Why does it need to re-render everything again? just set the attribute it might be better

Reply:
It will not re-render, `lit` engine checks if it needs to re-render and checks how to re-render, in this particular case it would update the attribute. It is easy to track the flow of updates if all updates are happening at one place, in this case in `render` method. And another reason, that it is more performant to make all updates in batched update. Please read here: https://lit.dev/docs/components/lifecycle/#reactive-update-cycle on why this is important.

Reply:
I get it, this is better.